_posts/2022-12-14-OLS-Fellowships.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: post
+title: "OLS Fellowships: Announcing our next generation of community changemakers"
+authors: 
+- yochannah
+- pherterich
+- msundukova
+image: https://images.unsplash.com/photo-1516381548400-349d680edb56?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=735&q=80
+photos:
+  name: Photo by Jon Tyson on Unsplash
+  url: https://unsplash.com/photos/r9T0LZv8xWQ
+---
+
+# OLS Fellowships: Announcing our next generation of community changemakers
```

Comment:
The title here should be removed as it is already in the metadata 

![image](https://user-images.githubusercontent.com/1842467/207623919-370781c4-d693-4dbc-970a-939b0e76d8e5.png)

And it somehow looks strange in the blog post list: 

![image](https://user-images.githubusercontent.com/1842467/207624153-9771bf4e-be14-482e-bc2f-3f4d6ed5c1f4.png)

Reply:
Thanks - fixed in https://github.com/open-life-science/open-life-science.github.io/pull/531